modules/olm-bundle/00_mod.mk

```diff
@@ -0,0 +1,52 @@
+# Utility functions
+fatal_if_undefined = $(if $(findstring undefined,$(origin $1)),$(error $1 is not set))
+first_existing_file = $(firstword $(wildcard $1))
+first_existing_file_or_default = $(firstword $(wildcard $1) $(firstword $1))
+
+# Utility variables
+current_makefile := $(lastword $(MAKEFILE_LIST))
+current_makefile_directory := $(dir $(current_makefile))
+olm_base_dir := $(current_makefile_directory)/base
+
+# Default config
+olm_publish_repos ?= 
+olm_project_id ?=
+oci_olm_image_tag ?= $(VERSION)
+preflight_container_project_ids ?=
+
+# The example directory is used to find CRD examples, this is used to create the
+# alm-examples annotation in the clusterserviceversion
+olm_example_directory ?= deploy/examples
+
+# Base directory for OLM bundle
+olm_bundle_base := bundle
+olm_manifests_base := $(olm_bundle_base)/manifests
+olm_metadata_base := $(olm_bundle_base)/metadata
+olm_scorecard_base := $(olm_bundle_base)/tests/scorecard
+
+# We want to allow both the .yml and .yaml extension, so we use wildcard to find 
```

Comment:
@ThatsMrTalbot why do we support the .yml extension also?

Reply:
In modules/olm-bundle/base/bundle/README.md you explicitly state that the files have to be `$(deploy_name).clusterserviceversion.yaml` `$(olm_metadata_base)/annotation.yaml` etc.
That will also simplify this part of the makefile a bit.